src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationRepositoryTests.cs

```diff
@@ -12,21 +12,102 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
 {
     public class PublicationRepositoryTests
     {
+        [Fact]
+        public async void ReleasesCorrectlyOrdered()
+        {
+            var topicId = Guid.NewGuid();
+
+            using (var context = InMemoryApplicationDbContext("ReleasesCorrectlyOrdered"))
```

Comment:
This test file doesn't really demonstrate this, but rather than provide an explicit name, we should provide a `contextId` which is assigned to a `Guid.NewGuid().ToString()`.

This is less brittle to the method name changing, and also means you don't need to come up with a new unique name for every single test method.

I would suggest updating the rest of the tests to do this as well.